<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataDownloaderUI</class>
 <widget class="QWidget" name="DataDownloaderUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>331</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>395</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>395</width>
    <height>333</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>데이터 다운로드</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_dbSelect">
           <property name="text">
            <string>DB 선택</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="dbSelectCombo">
           <item>
            <property name="text">
             <string>운영 DB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>테스트 DB</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLineEdit" name="snStartEdit">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>SN 시작 (예: 2310IL0001)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="snEndEdit">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>끝 번호 (예: 15)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="label_startDateEdit">
           <property name="text">
            <string>시작일</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="label_endDateEdit">
           <property name="text">
            <string>종료일</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkAllBox">
       <property name="text">
        <string>전체 선택/해제</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="sensorGrid">
       <item row="0" column="0">
        <widget class="QCheckBox" name="check_PM25">
         <property name="text">
          <string>PM2.5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="check_PM10">
         <property name="text">
          <string>PM10</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="check_TEMP">
         <property name="text">
          <string>TEMP</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="check_HUMI">
         <property name="text">
          <string>HUMI</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="check_HCHO">
         <property name="text">
          <string>HCHO</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="check_NOISE">
         <property name="text">
          <string>NOISE</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="check_CO2">
         <property name="text">
          <string>CO2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="check_CO">
         <property name="text">
          <string>CO</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="check_VOC">
         <property name="text">
          <string>VOC</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="check_NO2">
         <property name="text">
          <string>NO2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>다운로드</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
